integration-templates/xero/actions/xero-update-invoice.ts

```diff
@@ -0,0 +1,122 @@
+import type { NangoAction, InvoiceActionResponse, Invoice, FailedInvoice, ActionErrorResponse } from '../../models';
+import { getTenantId } from '../helpers/get-tenant-id.js';
+import { toInvoice } from '../mappers/to-invoice.js';
+
+export default async function runAction(nango: NangoAction, input: Invoice[]): Promise<InvoiceActionResponse> {
+    const tenant_id = await getTenantId(nango);
+
+    // Validate the invoices:
+
+    // 1) Invoice id is required
+    const invalidInvoices = input.filter((x: any) => !x.id);
+    if (invalidInvoices.length > 0) {
+        throw new nango.ActionError<ActionErrorResponse>({
+            message: `The invoice id is required to update the invoice.\nInvalid invoices:\n${JSON.stringify(invalidInvoices, null, 4)}`
+        });
+    }
+
+    const config = {
```

Comment:
could help to type those object (for autocompletion and error at the right place)
`const config: ProxyConfiguration`